groups/cdn/version

```diff
@@ -0,0 +1 @@
+cdn-1.0
```

Comment:
You'll need to remove the prefix here as the pipeline isn't configured to use a prefix for version tags (it's also unnecessary for this project as only a single semantic version tag will be used—`x.y.z`):

```suggestion
1.0
```